### PR TITLE
finish complete function, button, and routes

### PR DESCRIPTION
**Pull request by kousagi1012**

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -2,19 +2,32 @@ var express = require('express');
 var router = express.Router();
 const db = require("../database/database")
 /* GET home page. */
-router.get('/', function(request, response, next) { //if u go to home of where ur at, it sends req to server
-  db.getTask().then(task => { //our request is the function db.getTask and then, it returns the promise of the responserun the function on the database, (promise is respones to render database)
-    response.render('index', {task}); //which is to render the index.pug in addition to tasks from our db
+
+router.get('/', function(request, response, next) {
```
Comment (shakalee14) [inline routes/index.js]: you can switch to using es2015 for example: 

```  
+router.post('/send', (request, response, next) => {
 +  const task = request.body.task
     db.createTask(task).then(() => {
       response.redirect('/')
    })
  })
```

File: routes/index.js [hunk 2]
```diff
@@ -2,19 +2,32 @@ var express = require('express');
...
-  console.log (request.body.task)
+router.post('/send', function(request, response, next) {
+  const task = request.body.task
    db.createTask(task).then(() => {
      response.redirect('/')
   })
 })
 
+router.post('/modify', function(request, response, next) {
+  const {del} = request.body
```
Comment (shakalee14) [inline routes/index.js]: nice!

File: views/layout.pug [hunk 3]
```diff
@@ -5,3 +5,5 @@ html
     link(rel='stylesheet', href='/stylesheets/style.css')
   body
     block content
+      script(src="https://code.jquery.com/jquery-3.1.1.min.js" integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8=" crossorigin="anonymous")
```
Comment (shakalee14) [inline views/layout.pug]: are you using jquery on this project?

Comment (kousagi1012) [inline views/layout.pug]: We were going to add bootstrap which requires jquery, but I'm not sure we will get to it.

**Pull request merged**